modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -78,7 +78,21 @@ above will no longer run in IE11 and will fail to bootstrap with an error and/or
 
 ### React 18 Upgrade
 
-> Coming Soon (https://github.com/Workday/canvas-kit/discussions/1545)
+In this release we upgraded React to version 18. [React 18](https://reactjs.org/blog/2022/03/29/react-v18.html) introduced several new features, but the
+most impactful change is the new [concurrent render API](https://reactjs.org/blog/2022/03/29/react-v18.html#what-is-concurrent-react). In the new API, you'll use the `.createRoot()`
+method which replaces `.render()`, which adds all of the improvements of React 18 and allows you to
+use concurrent features. Without it React behaves the same as it did in version 17.
+
+Canvas Kit doesn’t use the render API directly in our packages but Storybook uses it to render
+Canvas Kit components. Because of this, we upgraded Storybook dependencies to support React 18.
+
+**Impact on CK:** Updating to React 18 didn't cause any breaking changes in our code. However, we did have to update some types based on some type packages that where upgraded.
+To see React v18 features in action we had to upgrade Storybook to v6.5.x.
+
+**Impact on users:** Canvas Kit on React 18 doesn't produce any problem if it runs on < 18 version
+react environment. However, users will see a  console warning about `.root()` being deprecated. It's possible for
+users to use Canvas Kit v8 without upgrading to React 18 but some features may be ignored and limited
+by the legacy root API usage.
```

Comment:
I liked how the official announcement was clear about the React 18 upgrade not impacting users.

> Canvas Kit will continue to support React 16.14, React 17.X, and React 18.X  in v8.
>
> This change will not impact product teams that are already on v7 and are looking to upgrade to v8 as v8 will support the same React versions as v7 while still giving teams the ability to upgrade to React 18 as needed.

What you do think about reusing that language here and integrating it with what you've already written regarding the console warning and the limitations of using Canvas Kit v8 without React 18?

Also, can we be more specific about which features "may be ignored and limited by the legacy root API usage"? I assume Canvas Kit will work just the same for users who are still on React 16.14 or 17 -- are we referring to features which may be added to Canvas Kit in the future?